packages/engine/lib/execution/src/runner/javascript/mod.rs

```diff
@@ -284,13 +297,22 @@ fn module_resolve_callback<'s>(
     // Should be safe to ignore for now as an unsupported Javascript feature. https://v8.dev/features/import-assertions
     _import_assertions: v8::Local<'s, v8::FixedArray>,
     // Not used for now, a description can be found at https://github.com/denoland/rusty_v8/blob/25dd770570b77dfaa5a27a7dcee6fa660781640f/src/isolate.rs#L109
-    _referrer: v8::Local<'s, v8::Module>,
+    referrer: v8::Local<'s, v8::Module>,
 ) -> Option<v8::Local<'s, v8::Module>> {
     // SAFETY: we are in a callback
     let mut scope = unsafe { v8::CallbackScope::new(context) };
     let specifier = specifier.to_rust_string_lossy(&mut scope);
+    let global_referrer = v8::Global::new(&mut scope, referrer);
+    let referrer_path = scope
+        .get_slot::<Rc<RefCell<ModuleMap>>>()
+        .expect("ModuleMap is not present in isolate slots")
+        .borrow()
+        .path_by_modules
+        .get(&global_referrer)
+        .expect(&format!("Referrer to be already imported. {specifier}"))
+        .clone();
```

Comment:
This clone makes me a bit sad but this is the only way I could think of to stop the `RefCell` borrow.